File: wwwroot/js/dto/task/GetTaskByIdResponse.js
export class GetTaskByIdResponse {
  constructor({ id, title, description, dueDate, statusId, createdBy, createdAt, updatedAt, assignedUsers = [] }) {
    this.id = id;
    this.title = title;
    this.description = description;
    this.dueDate = dueDate;
    this.statusId = statusId;
    this.createdBy = createdBy;
    this.createdAt = createdAt;
    this.updatedAt = updatedAt;
    this.assignedUsers = assignedUsers;
  }
}

File: Source/Infrastructure/Adapters/Input/DTOs/Task/GetTaskByIdResponse.cs
using TaskBoardDemo.Source.Infrastructure.Adapters.Input.DTOs.User;

namespace TaskBoardDemo.Source.Infrastructure.Adapters.Input.DTOs.Task;

public class GetTaskByIdResponse
{
    public Guid Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string? Description { get; set; }
    public DateTime? DueDate { get; set; }
    public Guid StatusId { get; set; }
    public Guid CreatedBy { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public List<GetUserByIdResponse> AssignedUsers { get; set; } = new List<GetUserByIdResponse>();
}
